badge_server/test_badges.py

```diff
@@ -522,6 +542,16 @@ def assertBadgeStatusToColor(self, badge_status_to_color):
 class TestSuccess(BadgeTestCase):
     """Tests for the cases where the badge image displays 'success.'"""
 
+    def assertImageResponsePyPI(self, package_name):
```

Comment:
I'd prefer not to bask a base class method with one with a different signature.

Reply:
I thought that the function names were getting long/convoluted, especially since by looking at the test suite each method is being implemented in, it should be clear what the expected status would be.
(eg `assertImageObsoleteDependencyPyPI()` and `assertImageSelfIncompatibleGithub()`) 

What if I pull `_assertImageResponsePyPI()` / `_assertImageResponseGithub()` from the base class and have other classes implement `assertImageResponsePyPI()` / `assertImageResponseGithub()`?

Reply:
OK, let's just leave it the way that it is then.